### PR TITLE
Add minimalAmount for Ramp provider

### PR DESCRIPTION
**Pull request by cehali**
# Summary

### CODE REVIEW

File: universal-login-react/src/core/constants/normalization.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const GET_PRICE_IN_ETHER_NORMALIZE_TO_INTEGER = 100;
+
+export const GET_PRICE_IN_ETHER_NORMALIZE_TO_UNITS = Math.pow(10, 18).toString();
```
Comment (xitronix) [inline universal-login-react/src/core/constants/normalization.ts]: Can inline: ethers/constants => WeiPerEther.toString()

File: universal-login-react/src/core/utils/getMinimalAmountForFiatProvider.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {utils} from 'ethers';
+import {getEtherPriceInCurrency} from '@universal-login/sdk';
+import {TopUpProvider} from '../../core/models/TopUpProvider';
+import {getPriceInEther} from './getPriceInEther';
+
+export const getMinimalAmountForFiatProvider = async (paymentMethod: TopUpProvider, requiredDeploymentBalance: string) => {
+  switch (paymentMethod) {
+    case 'RAMP': {
+      const providerMinimalAmountInFiat = '1';
+      const etherPriceInGBP = await getEtherPriceInCurrency('GBP');
```
Comment (xitronix) [inline universal-login-react/src/core/utils/getMinimalAmountForFiatProvider.ts]: Proposal: convertToEther

File: universal-login-react/src/core/utils/getMinimalAmountForFiatProvider.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      const providerMinimalAmount = getPriceInEther(providerMinimalAmountInFiat, etherPriceInGBP);
+      const requiredDeploymentBalanceAsBigNumber = utils.parseEther(requiredDeploymentBalance);
+      return requiredDeploymentBalanceAsBigNumber.gt(providerMinimalAmount)
+        ? utils.formatEther(requiredDeploymentBalanceAsBigNumber)
+        : utils.formatEther(providerMinimalAmount);
```
Comment (xitronix) [inline universal-login-react/src/core/utils/getMinimalAmountForFiatProvider.ts]:
```ts
return utils.formatEther(bigNumberMax(
     requiredDeploymentBalanceAsBigNumber,
     providerMinimalAmount
));

```

File: universal-login-react/src/core/utils/getPriceInEther.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import {utils} from 'ethers';
+import {GET_PRICE_IN_ETHER_NORMALIZE_TO_INTEGER, GET_PRICE_IN_ETHER_NORMALIZE_TO_UNITS} from '../constants/normalization';
+
+export const getPriceInEther = (priceInFiat: string, etherPriceInFiat: string): utils.BigNumber => {
+  const etherPriceInFiatAsBigNumber = utils.parseEther(etherPriceInFiat);
+  const etherPriceNormalizedToInteger = etherPriceInFiatAsBigNumber.mul(GET_PRICE_IN_ETHER_NORMALIZE_TO_INTEGER);
+  const etherPriceNormalizedToUnits = etherPriceNormalizedToInteger.div(GET_PRICE_IN_ETHER_NORMALIZE_TO_UNITS);
+  const priceInFiatAsWei = utils.parseEther(priceInFiat);
+  const priceInFiatNormalized = priceInFiatAsWei.mul(GET_PRICE_IN_ETHER_NORMALIZE_TO_INTEGER);
```
Comment (xitronix) [inline universal-login-react/src/core/utils/getPriceInEther.ts]: Consider to use: ethers/utils:
```ts
parse(value, someDecimal)
```

File: universal-login-react/src/core/constants/normalization.ts (hunk 1 above)
Comment (cehali) [reply universal-login-react/src/core/constants/normalization.ts]: 👍

File: universal-login-react/src/core/utils/getMinimalAmountForFiatProvider.ts (hunk 3 above)
Comment (cehali) [reply universal-login-react/src/core/utils/getMinimalAmountForFiatProvider.ts]: 👍

File: universal-login-react/src/core/utils/getPriceInEther.ts (hunk 4 above)
Comment (cehali) [reply universal-login-react/src/core/utils/getPriceInEther.ts]: 👍

File: universal-login-react/src/core/utils/getPriceInEther.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import {utils} from 'ethers';
+import {WeiPerEther} from 'ethers/constants';
+import {GET_PRICE_IN_ETHER_NORMALIZE_WEI_INTEGER} from '../constants/normalization';
+
+const {parseUnits} = utils;
+
+export const getPriceInEther = (priceInFiat: string, etherPriceInFiat: string): utils.BigNumber => {
+  const etherPriceNormalizedToInteger = parseUnits(etherPriceInFiat, GET_PRICE_IN_ETHER_NORMALIZE_WEI_INTEGER);
```
Comment (xitronix) [inline universal-login-react/src/core/utils/getPriceInEther.ts]: Inline GET_PRICE_IN_ETHER_NORMALIZE_WEI_INTEGER or create private( no export) const in this file
Rename  GET_PRICE_IN_ETHER_NORMALIZE_WEI_INTEGER => DECIMAL_FOR_NORMALIZE

**Pull request merged**